var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -621,6 +621,17 @@ def link_debug_libs(self):
                     libname = os.path.split(lib)[1]
                     os.symlink(libname, libname.replace("_debug", ""))
 
+    @run_after("install")
+    def symlink_to_h5hl_wrappers(self):
+        if self.spec.satisfies("+hl"):
+            with working_dir(self.prefix.bin):
+                # CMake's FindHDF5 relies only on h5cc so it doesn't find the HL
+                # component unless it uses h5hlcc so we symlink h5cc to h5hlcc etc
+                os.remove("h5cc")
+                os.remove("h5c++")
+                symlink("h5hlcc", "h5cc")
+                symlink("h5hlc++", "h5c++")
```

Comment:
```suggestion
                symlink_files = {
                                  "h5cc":  "h5lcc",
                                  "h5c++": "h5lc++"
                 }
                 for old, new in symlink_files.items():
                     if os.path.isfile(old):
                         os.remove(old)
                         symlink(new, old)
```
Looks like this is failing the CI pipelines b/c these files aren't always there? This suggestion will hopefully resolve that, but it is likely a version issue that needs to be satisfied along with the `+hl`?